package-lock.json

```diff
@@ -1,4 +1,5 @@
 {
+	"name": "jsonforms-edgar",
```

Comment:
Where is `jsonforms-edgar` coming from?

To generate a clean `package-lock` please delete your existing one and execute `rm package-lock.json && git clean -dfx && npm install && git clean -dfx && npm ci`.